ajax/libs/geomicons-open/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "geomicons-open",
+  "filename": "geomicons.min.js",
```

Comment:
I don't see this file in the npm package?

```
tree node_modules/geomicons-open

node_modules/geomicons-open
├── CNAME
├── README.md
├── build-svg.js
├── dist
│   └── svg
│       ├── bookmark.svg
│       ├── calendar.svg
│       ├── camera.svg
│       ├── chat.svg
│       ├── check.svg
│       ├── chevronDown.svg
│       ├── chevronLeft.svg
│       ├── chevronRight.svg
│       ├── chevronUp.svg
│       ├── clock.svg
│       ├── close.svg
│       ├── cloud.svg
│       ├── cog.svg
│       ├── compose.svg
│       ├── expand.svg
│       ├── external.svg
│       ├── file.svg
│       ├── folder.svg
│       ├── geolocation.svg
│       ├── grid.svg
│       ├── heart.svg
│       ├── home.svg
│       ├── info.svg
│       ├── link.svg
│       ├── list.svg
│       ├── lock.svg
│       ├── mail.svg
│       ├── musicNote.svg
│       ├── next.svg
│       ├── no.svg
│       ├── pause.svg
│       ├── picture.svg
│       ├── pin.svg
│       ├── play.svg
│       ├── previous.svg
│       ├── refresh.svg
│       ├── repost.svg
│       ├── search.svg
│       ├── shoppingCart.svg
│       ├── skull.svg
│       ├── speaker.svg
│       ├── speakerVolume.svg
│       ├── star.svg
│       ├── tag.svg
│       ├── trash.svg
│       ├── triangleDown.svg
│       ├── triangleLeft.svg
│       ├── triangleRight.svg
│       ├── triangleUp.svg
│       ├── user.svg
│       ├── video.svg
│       └── warning.svg
├── package.json
├── src
│   ├── index.js
│   └── paths
│       ├── bookmark.js
│       ├── calendar.js
│       ├── camera.js
│       ├── chat.js
│       ├── check.js
│       ├── chevron-down.js
│       ├── chevron-left.js
│       ├── chevron-right.js
│       ├── chevron-up.js
│       ├── clock.js
│       ├── close.js
│       ├── cloud.js
│       ├── cog.js
│       ├── compose.js
│       ├── expand.js
│       ├── external.js
│       ├── file.js
│       ├── folder.js
│       ├── geolocation.js
│       ├── grid.js
│       ├── heart.js
│       ├── home.js
│       ├── info.js
│       ├── link.js
│       ├── list.js
│       ├── lock.js
│       ├── mail.js
│       ├── music-note.js
│       ├── next.js
│       ├── no.js
│       ├── pause.js
│       ├── picture.js
│       ├── pin.js
│       ├── play.js
│       ├── previous.js
│       ├── refresh.js
│       ├── repost.js
│       ├── search.js
│       ├── shopping-cart.js
│       ├── skull.js
│       ├── speaker-volume.js
│       ├── speaker.js
│       ├── star.js
│       ├── tag.js
│       ├── trash.js
│       ├── triangle-down.js
│       ├── triangle-left.js
│       ├── triangle-right.js
│       ├── triangle-up.js
│       ├── user.js
│       ├── video.js
│       └── warning.js
└── test
    └── index.js

5 directories, 110 files
```

Reply:
I think there is an issue with npm(yarn) install  for latest beta version. Cause of you check [here](https://libraries.io/npm/geomicons-open/3.0.0-beta.2) the file is exists in dist folder

Reply:
So, now I need to add also src folder to glob and use index.js?

Reply:
Sorry for the slow reply, does `geomicons.min.js` exist in the old versions?
If so, I think it'll actually be fine to leave this as-is. I would say that this library doesn't have a main file anymore :)

Reply:
The versions from 2.0.0 and below have this file